corehq/messaging/smsbackends/vertex/models.py

```diff
@@ -87,7 +88,7 @@ def handle_failure(self, msg_obj, resp_status_code, error_message):
             error_message in GATEWAY_ERROR_MESSAGES
         ):
             msg_obj.set_system_error(SMS.ERROR_TOO_MANY_UNSUCCESSFUL_ATTEMPTS)
-            raise VertexBackendException(error_message)
+            notify_exception(None, error_message)
```

Comment:
would be good to also prepend something like "Error with the Vertex SMS Backend: "